src/app/main.c

```diff
@@ -118,6 +118,8 @@ static RpmOstreeCommand commands[] = {
                     RPM_OSTREE_BUILTIN_FLAG_REQUIRES_ROOT |
                     RPM_OSTREE_BUILTIN_FLAG_HIDDEN,
     NULL, rpmostree_builtin_start_daemon },
+  { "finalize-deployment", RPM_OSTREE_BUILTIN_FLAG_HIDDEN,
```

Comment:
Do we want to bake this in `ex` first? I'm fine either way, as the client consuming this will be version-locked to rpm-ostree anyway.

Reply:
Yeah, I initially thought of adding it in `ex`, but decided not to given that (1) it's hidden and not meant to be user-facing, and (2) the corresponding switch is in `upgrade`.